src/blueapi/cli/cli.py

```diff
@@ -116,7 +118,7 @@ def wrapper(*args, **kwargs):
 def get_plans(obj: dict) -> None:
     """Get a list of plans available for the worker to use"""
     client: BlueapiRestClient = obj["rest_client"]
-    pprint(client.get_plans().dict())
+    print(client.get_plans().dict())
```

Comment:
Dictionaries should still be `pprint`s, ditto for the other pprints

Reply:
ok sure

Reply:
changed all dict prints into pprints